### PR TITLE
min execution premium for MM

### PR DESCRIPTION
**Pull request by DonDuala**
fixes https://github.com/Dual-Finance/risk-manager/issues/94

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -90,6 +91,19 @@ export class Router {
...
+          this.dip_to_string(
+            dip_deposit.expirationMs / 1_000,
+            dip_deposit.strike,
+          )
+        ] = dip_deposit;
+        this.run_risk_manager();
+        return;
+      }
       // TODO: Test this to make sure the decimals are correct on each.
       console.log('MM price:', price, 'BVE price:', thresholdPrice);
```
Comment (brittcyr) [inline src/router.ts:108]: move the log line above your block of code

**Review by brittcyr — APPROVED** [review]

File: src/router.ts (hunk 1 above)
Comment (DonDuala) [reply src/router.ts:108]: 96 or 107?

Comment (brittcyr) [reply src/router.ts:108]: console.log('MM price:', price, 'BVE price:', thresholdPrice);

Comment (brittcyr) [reply src/router.ts:108]: want the mm price to be logged before the first time we make a decision based on it

**Pull request merged**